src/test/ui/suggestions/issue-86100-tuple-paren-comma.stderr

```diff
@@ -49,6 +49,8 @@ LL |     let _s = S { _s: ("abc".to_string(),) };
 error[E0308]: mismatched types
   --> $DIR/issue-86100-tuple-paren-comma.rs:23:22
    |
+LL |     let t = (1, 2);
```

Comment:
I'm not exactly sure how to express this, but I'd prefer if we didn't emit this for literal expressions or expression that are almost "literal-like" -- this suggestion doesn't seem to add much value, but I'm not exactly sure if I can exactly figure why.

Reply:
I do think that there's some value to *show* the code where the initialization took place, so that the user doesn't have to go immediately back to their editor to understand what happened. Having said that, I'm ok eliding some of these if you feel that's a better idea.

Reply:
Hm, I just feel like this PR is stated to do something different (showing where an inferece type is *refined* to a specific value) but then also increases verbosity in a separate situation (pointing out where a local is initialized) as a side-effect.

I think if we *do* want to point out where a local is initialized in more cases, that should be done separately, so we can properly gauge its impact and maybe have a deeper discussion about it.